.github/workflows/ci.yml

```
@@ -54,6 +54,9 @@ jobs:
           - ember-lts-3.24
           - ember-lts-3.28
           - ember-lts-4.4
+          - ember-lts-4.8
```

not sure what the value of keeping 4.4 around is

---

my thought process is that it should be each LTS from oldest supported

---

ci isn't that slow now and this only adds some extra work which is what I'm concerned about, but it's probably fine

---

I prefer adding the latest embroider to it
```suggestion
          - ember-lts-4.12
          - ember-lts-5.4
          - ember-release
          - ember-beta
          - ember-canary
          - embroider-safe
          - embroider-optimized
```

---

@villander v5 and embroider are omit intentionally, until we get #855 over the finish line

---

@SergeAstapov can we get the finish line before change the pipeline? I’m pretty sure it has more priority than internal things

---

855 adds the extra pipelines you're talking about @villander, this bit is fine as is

---

@void-mAlex thanks for the clarifications